src/pages/Main/Main.js

```diff
@@ -1,5 +1,66 @@
+import { useState, useEffect } from 'react';
+
+import Banner from './Banner/Banner';
+import Product from './Product/Product';
+
+import './Main.scss';
+
 function Main() {
-  return <div />;
+  const [productList, setProductList] = useState([]);
+
+  useEffect(() => {
+    fetch('http://10.58.3.222:8000/products/list', {
+      method: 'GET',
+    })
```

Comment:
```suggestion
    fetch('http://10.58.3.222:8000/products/list')
```
GET 요청을 보낼 때는 이렇게 아무것도 작성하지 않아도 기본 값에 따라 GET 요청으로 인식합니다.